app/model/editions/EditionsIssue.scala

```diff
@@ -17,7 +18,14 @@ case class EditionsIssue(
     launchedBy: Option[String],
     launchedEmail: Option[String],
     fronts: List[EditionsFront]
-)
+) {
+  def toPublishedIssue(): PublishedIssue = PublishedIssue(
```

Comment:
Not a big fan of this pattern of enshrining conversion logic into the model. Would prefer this sat in a conversion companion object somewhere.